Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -397,26 +481,39 @@ namespace gdjs {
         );
         return;
       }
-      const onDismissBanner = () => {
-        removeAuthenticationBanner(runtimeScene);
-      };
-      const onOpenAuthenticationWindow = () => {
-        openAuthenticationWindow(runtimeScene);
-      };
-      // We display the corresponding banner depending on the authentication status.
-      _authenticationBanner = _userToken
-        ? authComponents.computeAuthenticatedBanner(
-            onOpenAuthenticationWindow,
-            onDismissBanner,
-            _username
-          )
-        : authComponents.computeNotAuthenticatedBanner(
-            onOpenAuthenticationWindow,
-            onDismissBanner
-          );
+
+      _authenticationBanner = createAuthenticationBanner(runtimeScene);
       domElementContainer.appendChild(_authenticationBanner);
     };
 
+    /**
+     * Helper to recompute the authentication banner.
+     * This is useful if the user is already logged in on Liluo.io
+     * and we want to display the banner with the username.
+     */
+    const refreshAuthenticationBanner = function (
```

Comment:
Ubernitpicking, to show that this is not necessarily refreshed if there is no banner shown:

```suggestion
    const refreshAuthenticationBannerIfAny = function (
```